src/containers/pages/GenericJurisdictionReport/index.tsx

```diff
@@ -197,14 +199,9 @@ const GenericJurisdictionReport = (
     currentJurisdictionName = theObject[0].jurisdiction_name;
   }
 
-  const currLevelData = data.filter(el => el.jurisdiction_parent_id === jurisdictionId);
-
-  let columnsToUse = get(plansTableColumns, jurisdictionColumn, null);
-  if (currLevelData && currLevelData.length > 0) {
-    if (currLevelData[0].jurisdiction_depth >= +focusAreaLevel) {
-      columnsToUse = get(plansTableColumns, focusAreaColumn, null);
-    }
-  }
+  const columnsToUse = customColumnsGet
```

Comment:
Are you avoiding getting this from props to maintain the way it currently works?

Reply:
not really, my thought was we have the way it works as default and if one intends to change that is when a prop is passed to override

Reply:
why not have `getColumnsToUse` as the default value of the prop?